StatsdClient/StatsdClientExtensions.cs

```diff
@@ -24,7 +31,14 @@ public static void LogCount(this IStatsd client, string name, int count = 1)
         /// <param name="milliseconds">The duration, in milliseconds, for this metric.</param>
         public static void LogTiming(this IStatsd client, string name, long milliseconds)
         {
-            client.LogTimingAsync(name, milliseconds).Wait();
+            try
+            {
```

Comment:
Can you revert this change, the aggregate exception should be boiled up as it can contain multiple exceptions.